src/main/java/Tasks/Event.java

```diff
@@ -0,0 +1,83 @@
+package Tasks;
+
+import Exceptions.DukeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an Event that will happen. An <code>Event</code> object
+ * has a <code>LocalDateTime</code> start and end time.
+ */
+public class Event extends Task {
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+    public Event(String description) throws DukeException {
```

Comment:
Perhaps adding Javadocs for your methods will help readers be able to better understand the functionality of your methods. Do consider adding it!